app/components/file_upload_preview/file_upload_item/file_upload_item.js

```diff
@@ -145,6 +163,21 @@ export default class FileUploadItem extends PureComponent {
         this.uploadPromise.then(this.handleUploadCompleted).catch(this.handleUploadError);
     };
 
+    downloadFile = async (file) => {
+        const options = {fileCache: true};
+        const tempDownloadFile = await RNFetchBlob.config(options).fetch('GET', file.localPath);
+        return tempDownloadFile;
+    }
+
+    cleanUpTempFile = () => {
+        if (!this.tempDownloadFile) {
+            return;
+        }
+
+        this.tempDownloadFile.flush();
```

Comment:
if we flush the tempfile when you open the app again the file won't be there so it will not be rendered, by saving the file as I mentioned above there is no need to flush it.

Reply:
hi @enahum since there is a cache mechanism, i have changed to use it to download the image instead https://github.com/mattermost/mattermost-mobile/pull/3084/commits/0ee38a0e9668db75a07305da8320309f2ec0668a